Trinap/Sources/Presenter/TabBar/Main/PhotographerList/PhotographerListViewController.swift

```diff
@@ -50,26 +48,34 @@ final class PhotographerListViewController: BaseViewController {
     // MARK: - Methods
     override func viewDidLoad() {
         super.viewDidLoad()
-        navigationController?.navigationBar.topItem?.titleView = searchBar
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(false)
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+    
     override func configureHierarchy() {
         self.view.addSubviews([
-            searchBar,
+            searchBarView,
             filterView,
             collectionView
         ])
     }
 
     override func configureConstraints() {
-        searchBar.snp.makeConstraints { make in
-            make.top.equalToSuperview()
-        }
         
-        filterView.snp.makeConstraints { make in
+        searchBarView.snp.makeConstraints { make in
+            make.top.equalTo(view.safeAreaLayoutGuide)
             make.leading.equalToSuperview().offset(trinapOffset)
             make.trailing.equalToSuperview().offset(-trinapOffset)
-            make.top.equalTo(view.safeAreaLayoutGuide)
+            make.height.equalTo(48)
+        }
+        
+        filterView.snp.makeConstraints { make in
+            make.leading.equalToSuperview()
+            make.trailing.equalToSuperview()
```

Comment:
```suggestion
            make.horizontalEdges.equalToSuperview()
```
요렇게 한 줄로 수정해도 좋을 것 같습니다!